app/frontend/components/common/nufiBanner.tsx

```diff
@@ -1,14 +1,26 @@
 import {h} from 'preact'
 
-const NufiBanner = () => {
+const NufiBanner = ({staticImage = true}: {staticImage?: boolean}) => {
```

Comment:
I'd rather be crystal clear here
```suggestion
const NufiBanner = ({variant = 'static'}: {variant?: 'static' | 'gif'}) => {
```
Optionally I'd even remove the parameter optionality so that it's clear from the caller which variant is rendered